src/Illuminate/Routing/Router.php

```diff
@@ -674,6 +677,21 @@ public static function prepareResponse($request, $response)
     }
 
     /**
+     * Create an illuminate response from a given base response object.
+     *
+     * @param  mixed  $response
+     * @return \Illuminate\Http\Response
+     */
+    private static function createResponseFromBase($response)
+    {
+        return (new Response(
+            $response->getContent(),
```

Comment:
I think the result of conversion to JSON in `JsonResponse` is different than in `Response` so I imagine using the final output of the original response as the output of the new response is better, at the end of the day I think it's an edge case that you can easily get around by using the baseResponse.

Maybe I'm confused and there are more use cases, but I don't think reconverting the original content inside `Response` makes sense.